aws/client/client.go

```diff
@@ -89,7 +89,10 @@ const logReqMsg = `DEBUG: Request %s/%s Details:
 
 func logRequest(r *request.Request) {
 	logBody := r.Config.LogLevel.Matches(aws.LogDebugWithHTTPBody)
-	dumpedBody, _ := httputil.DumpRequestOut(r.HTTPRequest, logBody)
+	dumpedBody, err := httputil.DumpRequestOut(r.HTTPRequest, logBody)
+	if err != nil {
+		r.Config.Logger.Log(fmt.Sprintf("Error dumping request: %s", err))
+	}
```

Comment:
Do we want to continue on logging the `dumpedBody`, if an error was thrown? It is probably an empty string. This would make the log after this one pretty much useless.